pkg/common/ratelimiter.go

```diff
@@ -26,11 +29,14 @@ func NewDefaultRateLimiter() RateLimiter {
 	return RateLimiter{rateLimiters: map[Event]*rate.Limiter{
 		// Below are the default limits for the AWS CloudMap's APIs
 		// TODO: make it customizable in the future
-		ListNamespaces:     rate.NewLimiter(rate.Limit(1), 5),     // 1 ListNamespaces API calls per second
-		ListServices:       rate.NewLimiter(rate.Limit(2), 10),    // 2 ListServices API calls per second
-		GetOperation:       rate.NewLimiter(rate.Limit(100), 200), // 100 GetOperation API calls per second
-		RegisterInstance:   rate.NewLimiter(rate.Limit(50), 100),  // 50 RegisterInstance API calls per second
-		DeregisterInstance: rate.NewLimiter(rate.Limit(50), 100),  // 50 DeregisterInstance API calls per second
+		ListNamespaces:      rate.NewLimiter(rate.Limit(0.5), 5),    // 1 ListNamespaces API calls per second
```

Comment:
Comment should be 0.5tps?

Reply:
Will fix this in the follow-up task I'm working on.